chesster.js

```diff
@@ -1401,3 +1427,179 @@ function result_reply_missing_pairing(bot, message){
 function result_reply_updated(bot, message, result){
     bot.reply(message, "Got it. @" + result.white.name + " " + result.result + " @" + result.black.name);
 }
+
+
+
+/* game link parsing */
+
+//given a gamelink_id, use the lichess api to get the game details
+//pass the details to the callback as a JSON object
+function fetch_game_details(gamelink_id, callback){
+    const http = require('http');
+    var url = "http://en.lichess.org/api/game/" + gamelink_id;
```

Comment:
I feel like we could make use of a function that just reads a result into a JSON Structure in memory. And this is the only line here that's not about that aspect.  I'd be in favour (but it's not required for this PR) to refactor this into a "fetch_url_into_json" method and to then pass in a complete url. 

you could keep a fetch_game_details method around that takes only a gamelink_id, but it should use the above method internally.